sc/social/like/tests/test_plugin_facebook.py

```diff
@@ -251,12 +251,12 @@ def test_plugin_language(self):
         self.document.setLanguage('pt-br')
         view = document.restrictedTraverse(plugin_view)
         html = view.metadata()
-        self.assertIn('connect.facebook.net/pt_BR/all.js', html)
+        self.assertIn('connect.facebook.net/pt_BR/sdk.js', html)
 
         self.document.setLanguage('en')
         view = document.restrictedTraverse(plugin_view)
         html = view.metadata()
-        self.assertIn('connect.facebook.net/en_GB/all.js', html)
+        self.assertIn('connect.facebook.net/en_GB/sdk.js', html)
```

Comment:
The code that produces the language is not a simple one.  I'll use this issue to check it out https://github.com/collective/sc.social.like/issues/122